src/components/tool/guide/EresearchJob/EresearchInstructions.js

```diff
@@ -46,6 +46,7 @@ export function EresearchInstructions({config}) {
                     gpuModel={config.gpuModel}
                     gpuModules={config.gpuModules}
                     nodes={config.nodes}
+                    wallTime={config.wallTime}
```

Comment:
Also need to update `LyraStartInstructions` in `HuggingFaceInstructions`

Reply:
Actually, not sure about this one

Reply:
Fixed